pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl

```diff
@@ -0,0 +1,136 @@
+import "HISAT2.wdl" as HISAT2
+import "Picard.wdl" as Picard
+import "RSEM.wdl" as RSEM
+
+workflow SmartSeq2SingleCell {
+  meta {
+    description: "Process SmartSeq2 scRNA-Seq data, include reads alignment, QC metrics collection, and gene expression quantitication"
+  }
+
+  # load annotation
+  File gtf_file
+  File genome_ref_fasta
+  File rrna_intervals
+  File gene_ref_flat
+
+  # load index
+  File hisat2_ref_index
+  File hisat2_ref_trans_index
+  File rsem_ref_index
+
+  # ref index name
+  File hisat2_ref_name
+  File hisat2_ref_trans_name
+
+  # samples
+  String stranded
+  String sample_name
+  String output_name
+  File fastq1
+  File fastq2
+
+  parameter_meta {
+    gtf_file: ""
+    genome_ref_fasta: ""
+    rrna_intervals: ""
+    gene_ref_flat: ""
+    hisat2_ref_index: ""
+    hisat2_ref_trans_index: ""
+    rsem_ref_index: ""
+    hisat2_ref_name: ""
+    hisat2_ref_trans_name: ""
+    stranded: "Library strand information example values: FR RF NONE"
+    sample_name: ""
+    output_name: ""
+    fastq1: "R1 in paired end reads"
+    fastq2: "R2 in paired end reads"
+  }
+
+  String quality_control_output_basename = output_name + "_qc"
```

Comment:
Should there be an empty line between String and call? I can't remember what we decided.

Reply:
good catch!